_data/v2_6/navbars/getting-started.yml

```diff
@@ -23,6 +23,8 @@ toc:
         path: /getting-started/kubernetes/installation/hosted/kubeadm/
       - title: Kubernetes Datastore Hosted Install
         path: /getting-started/kubernetes/installation/hosted/kubernetes-datastore/
+      - title: Canal Hosted Install
```

Comment:
Can we also include the word "flannel" because it may be more recognizable...we could just go with "Canal/flannel"